src/compiler/types.ts

```diff
@@ -109,6 +109,11 @@ export const enum SyntaxKind {
     BacktickToken,
     /** Only the JSDoc scanner produces HashToken. The normal scanner produces PrivateIdentifier. */
     HashToken,
+    /**
+     * Only the special JSDoc comment text scanner produces JSDocCommentTextTokes. One of these tokens spans all text after a tag comment's start and before the next @

```

Comment:
Just now noticing the parallel between this comment and the previous ones. Given people can call our scanner themselves, maybe I was wrong and this should be public so that no undocumented kinds are returned from `token`?